bin/bin.js

```diff
@@ -1,9 +1,7 @@
 #!/usr/bin/env node
 
 var run = require('..');
-var optimist = require('optimist');
-
-var argv = optimist
+var argv = require('yargs/yargs')(process.argv.slice(2))
```

Comment:
can you move the `require('yargs')` into a separate line? I like to keep requires and their use separated

Reply:
absolutely. done in b74c446